src/index.ts

```diff
@@ -502,6 +502,25 @@ export const types12: RegistryTypes = {
   },
 };
 
+export const types17: RegistryTypes = {
+  ...types12,
+  // Delegation updated types
+  DelegationNode: {
+    hierarchy_root_id: "DelegationNodeIdOf",
```

Comment:
```suggestion
    hierarchyRootId: "DelegationNodeIdOf",
```

camel case for JS

Reply:
Fixed in https://github.com/KILTprotocol/type-definitions/pull/10/commits/6ca305679bad9b60d57a94bab6f1c988c98894c6.